src/App.test.tsx

```diff
@@ -0,0 +1,109 @@
+jest.mock('./services/AuctionWinnerService');
+jest.mock('./services/BidGenerationService');
+import * as React from 'react';
+import { shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { DeterminWinningBidderAndPrice }  from './services/AuctionWinnerService';
+import { generateBids } from './services/BidGenerationService';
+
+import { App } from './App';
+
+// Setup Enzyme Adapter
+configure({ adapter: new Adapter() });
```

Comment:
Ideally this would be in `setupTests.ts`, but there is currently only one file running enzyme.